Sources/Ledger/TxOutUtils.swift

```diff
@@ -99,6 +99,51 @@ enum TxOutUtils {
         }
     }
 
+    static func decryptEMemoPayload(
```

Comment:
Not sure I'm crazy about the "EMemo" name here.. guessing the E is meant to stand for Encrypted, but it's not really necessary since the function name starts with decrypt, and the parameter name itself is "encryptedMemo"... 

Just trying to remove things that make people (like me lol) wonder "What's an EMemo?" when it's not really a type or a thing...

Reply:
Ahhh, I see the name ..._e_memo_... in the wrapped function call...  But still, if it's wrapping that call abstraction, no real need to percolate the name up to the swift function.

Reply:
i like EMemo its camel case for e_memo